kafka/src/main/java/zipkin2/reporter/kafka/KafkaSender.java

```diff
@@ -223,7 +223,7 @@ public KafkaSender build() {
    * <p>
    * See @{@link AdminClientConfig} config properties
    */
-  private Map<String, Object> filterPropertiesForAdminClient(Properties properties) {
+  protected Map<String, Object> filterPropertiesForAdminClient(Properties properties) {
```

Comment:
package private is better as the type is final

```suggestion
  Map<String, Object> filterPropertiesForAdminClient(Properties properties) {
```